lib/rzo/trollop.rb

```diff
@@ -0,0 +1,926 @@
+# lib/trollop.rb -- trollop command-line processing library
```

Comment:
Makes sense. Could you post a sentence at the top of the file that explains why this was vendored.